### PR TITLE
Backend: Browse Basic Page

### PR DESCRIPTION
**Pull request by kave**
Backend: Browse Basic Page

## Reviewers
- @anselmbradford 
- @KimberlyMunoz 
- @kurtw

### CODE REVIEW

File: cfgov/jinja2/v1/browse-basic/index.html [hunk 1]
```diff
@@ -30,100 +30,122 @@
...
-                        'is_expanded': true
-                    }
-                 })
-             %}
-            {% set value = global_dict.expandable %}
-            {% include 'molecules/expandable.html' with context %}
-            {% set value = None if prototype %}
+
+    {% for block in page.header -%}
+        <div class="block">
```
Comment (kave) [inline cfgov/jinja2/v1/browse-basic/index.html]: @anselmbradford so the wiki has 2 sections Area 1 & 2. do they need styling like the sub/landing pages?

Comment (anselmbradford) [inline cfgov/jinja2/v1/browse-basic/index.html]: Like the hero in the landing page do you mean? I'm not actually sure if they need to be visually separated or if area 1 & 2 are just defining where the organisms are clustered. @schaferjh do you know if there needs to be any styles added to visually separate area 1 & 2 on the /browse-basic/ mockup (other than those added by individual organisms)?

Comment (schaferjh) [inline cfgov/jinja2/v1/browse-basic/index.html]: You're right, @anselmbradford -- areas 1 and 2 are schematic indicators to help everyone understand hierarchy and organization of organisms within the page type. They do not need to be visually apparent to end users.

Comment (kave) [inline cfgov/jinja2/v1/browse-basic/index.html]: @schaferjh @anselmbradford great thanks clearing that up

File: cfgov/v1/models/browse_page.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+from wagtail.wagtailcore.models import PAGE_TEMPLATE_VAR
+
+from .base import CFGOVPage
+from . import molecules
+from . import organisms
+
+
+class BrowsePage(CFGOVPage):
+    header = StreamField([
+        ('text_introduction', molecules.TextIntroduction()),
```
Comment (kurtw) [inline cfgov/v1/models/browse_page.py]: Shouldn't there be a Featured Content molecule here too? Just comparing from the wiki image.

File: cfgov/v1/models/browse_page.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        ('image_text_25_75_group', organisms.ImageText2575Group()),
+        ('image_text_50_50_group', organisms.ImageText5050Group()),
+        ('half_width_link_blob_group', organisms.HalfWidthLinkBlobGroup()),
+        ('well', organisms.Well()),
+        ('full_width_text', organisms.FullWidthText()),
+        ('expandable_group', organisms.ExpandableGroup()),
+        ('table', organisms.Table()),
+    ], blank=True)
+
+    side_navigation = StreamField([], blank=True)
```
Comment (kurtw) [inline cfgov/v1/models/browse_page.py]: Say whaat?

File: cfgov/v1/models/browse_page.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    # General content tab
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        StreamFieldPanel('content'),
+    ]
+
+    sidebar_panels = [
+                         StreamFieldPanel('side_navigation'),
+                     ] + CFGOVPage.sidefoot_panels
```
Comment (kurtw) [inline cfgov/v1/models/browse_page.py]: Spacing issue.

File: cfgov/v1/models/browse_page.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    # Tab handler interface
+    edit_handler = TabbedInterface([
+        ObjectList(content_panels, heading='General Content'),
+        ObjectList(CFGOVPage.sidefoot_panels, heading='Sidebar'),
```
Comment (kurtw) [inline cfgov/v1/models/browse_page.py]: `CFGOVPage.sidefoot_panels` should be changed to `sidebar_panels`

File: cfgov/v1/models/browse_page.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+    ])
+
+    def get_context(self, request, *args, **kwargs):
+        return {
+            PAGE_TEMPLATE_VAR: self,
+            'self': self,
+            'request': request,
+        }
```
Comment (kurtw) [inline cfgov/v1/models/browse_page.py]: This is duplicate from the Wagtail code. https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailcore/models.py#L648

File: cfgov/v1/models/browse_page.py (hunk 2 above)
Comment (kave) [inline cfgov/v1/models/browse_page.py]: ah yes, i'll add that in

File: cfgov/v1/models/browse_page.py (hunk 3 above)
Comment (kave) [inline cfgov/v1/models/browse_page.py]: lol the elements for side navigation haven't been created yet. so we decided to create page models and include the elements piece-meal style as they get created

File: cfgov/v1/models/browse_page.py (hunk 6 above)
Comment (kave) [inline cfgov/v1/models/browse_page.py]: gotcha i'll remove it

**Comment by kurtw** [comment]
After adding the footer you get a :+1:

**Pull request merged**